### ISSUE TITLE
Migrate the AUR configuration and actions configuration back to the config-file-validator project

### ISSUE THREAD

**Issue by kehoecj** [Boeing/config-file-validator#176]
Migrate the code from the temporary [config-file-validator-aur](https://github.com/kehoecj/config-file-validator-aur) project back into this project. Once the code is ready you will need to create a draft MR and request that I create a shared secret for the project to be able to publish to AUR.

**Comment by timwehrle** [comment]
@kehoecj I can look into it.

**Comment by kehoecj** [comment]
@timwehrle Assigned it to you - let me know if you have any questions!

**Comment by timwehrle** [comment]
@kehoecj Should I also edit the release workflow? Because it would be easier if you did that and included the secrets. But I can do that too. 

Is there a way to automatically detect the release version of the latest release so we don't have to manually change that in the PGKBUILD file?

**Comment by kehoecj** [comment]
>Should I also edit the release workflow? Because it would be easier if you did that and included the secrets. But I can do that too.

Yes, please. Once your MR is up I'll add the secret and run it.

>s there a way to automatically detect the release version of the latest release so we don't have to manually change that in the PGKBUILD file?

If you want to modify the workflow to automatically update the PKGBUILD that'd be great

**Comment by timwehrle** [comment]
Okk, I'll check later.

**Comment by timwehrle** [comment]
@kehoecj You can check the PR.

**Comment by kehoecj** [comment]
@timwehr

> @kehoecj You can check the PR.

Merged. Feel free to open another PR for the dynamic PKGBUILD file work

**Comment by timwehrle** [comment]
@kehoecj Here is the second PR.

**Comment by kehoecj** [comment]
@timwehrle Thinking this can be closed. Agree?

**Comment by timwehrle** [comment]
@kehoecj Yea, sure. Go ahead 😄

**Comment by kehoecj** [comment]
@timwehrle Thanks again for your great work on this. Appreciate your support!
